package.json

```diff
@@ -6,10 +6,6 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1"
   },
-  "browserslist": [
-    "last 1 version",
-    "> 1%"
-  ],
```

Comment:
Hello, why this change?

Reply:
Moved to  seperate `.browserslistrc` file